### DISCUSSION TITLE
How to add button block in navigation block or background to navigation link?

### DISCUSSION THREAD

**Discussion by cawa-93** [WordPress/gutenberg#49027] [Q&A] [answered]
Sorry for the stupid question, but I can't figure out how to insert a button-like link into a navigation-block. I want to insert a button-block into the navigation-block, or change the background/text colors for the navigation-link-block.

`navigation-block` don't propose add button-block inside. Also `navigation-link-block` have only typografy settings but not color settings.

I started a new theme. Here my `theme.json`
```json
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 2,
	"settings": {
		"appearanceTools": true,

		"layout": {
			"contentSize": "1214px"
		}

	}
}
```

What I'm doing wrong?🤔

**Comment by talldan** [comment]
I may be wrong, but this might not be possible without using custom css and custom class names on the blocks you want to style.

There's still quite a lot of development work to do in terms of enabling users to customize the navigation block's appearance.

**Reply by cawa-93** [reply]
That sounds sad. A button (actually a button-like link) in the navigation, this is an extremely common design pattern. When there is a menu like: home, prices, contacts and a highlighted "BUY" link.

**Comment by skorasaurus** [comment, marked as answer]
This is not yet possible ; it has been requested in https://github.com/WordPress/gutenberg/issues/46256 ; you can follow its progress in that issue.
